Analyzers/src/AnalyzerCore.C

```diff
@@ -8,6 +8,7 @@ AnalyzerCore::AnalyzerCore(){
   cfEst = new CFBackgroundEstimator();
   pdfReweight = new PDFReweight();
   muonGE = new GeneralizedEndpoint();
+  muonGEScaleSyst = new GEScaleSyst();
```

Comment:
you don't delete the pointer in the destructor?

Reply:
fixed